import_handlers.js

```diff
@@ -0,0 +1,154 @@
+const fs = require('fs');
+const moment = require('moment');
+const xml2js = require('xml2js');
+//const logger = log4js.getLogger('<filename>');
+
+class Transaction {
+    constructor(date, from, to, narrative, amount) {
+        this.date = date;
+        this.from = from;
+        this.to = to;
+        this.narrative = narrative;
+        this.amount = parseFloat(amount);    
+    }
+}
+
+class Account {
+    constructor(name) {
+        this.name = name;
+        this.transactionRefs = [];
+    }
+
+    calcBalance() {
+        var running_total = 0;
+        let account_name = this.name;
+        this.transactionRefs.forEach(function(t) {
+            if (t.to === account_name) {

```

Comment:
I'd probably go for something along the lines of

    const toThisAccount = t.to === account_name;
    running_total += (toThisAccount ? 1 : -1) * t.amount;

But that's pretty subjective. I favour removing the redundancy of the amount + assignment between branches, and keeping the conditional to the smallest possible part of the code (i.e. whether we're moving the total up or down). I appreciate it's somewhat denser as a result.